app/marathon.py

```diff
@@ -1,9 +1,10 @@
 import socket
 
 class MarathonTaskInfo:
-	def __init__(self, host, ports):
+	def __init__(self, host, ports, labels):
```

Comment:
Yes. Since, we are reading all configs anyway. we could keep TaskInfo inside AppInfo. 
Not sure, if there is a way to read only a specific config of all apps.